resources/js/App/Components/Layout/Layout.jsx

```diff
@@ -0,0 +1,67 @@
+import React from "react";
+import "./layout.css"
+
+import Header from "../Header/Header.jsx";
+import LayoutLeft from "../LayoutLeft/LayoutLeft.jsx";
+import LayoutRight from "../LayoutRight/LayoutRight.jsx";
+
+
+export default class Layout extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      desktop: true,
+      width: window.innerWidth,
+      videoPageOn: false,
+    }
+  }
+
+  componentWillMount() {
+    if(this.state.width < 900) {
```

Comment:
This will work until user resizes the browser window. So, at least you also need to add an event listener for `resize` event and update `desktop` state property.

But much better solution would to not use JavaScript for this at all. You can do it purely with CSS using media queries. See docs: https://developer.mozilla.org/en-US/docs/Web/CSS/Media_Queries/Using_media_queries + there are plenty of examples on StackOverflow.